.github/workflows/linkcheck-reporter.yml

```diff
@@ -5,6 +5,10 @@ on:
     types:
       - completed
 
+permissions:
+  actions: read
+  pull-requests: write
```

Comment:
For future reference, based on my understanding: `actions: read` so it can download an output of the website validation workflow, and `pull-requests: write` so it can post comments on PRs.